frontend/packages/erd-core/src/features/gtm/utils/repositionTableLogEvent.ts

```diff
@@ -0,0 +1,12 @@
+import { pushToDataLayer } from './pushToDataLayer'
+
+type repositionTable = {
```

Comment:
[nits]
It seems better to capitalize the first letter of type definitions.

```suggestion
type RepositionTable = {
```